### PR TITLE
Adds update-notifier

### PR DESCRIPTION
**Pull request by reznord**
This adds update notifier to the CLI and reminds the user on every CLI command her runs. 

/cc @developit

### CODE REVIEW

**Comment by reznord** [comment]
Don't know how to test it locally lol 😛 

Hoping that it will work 😅

File: src/index.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ import installHooks from './lib/output-hooks';
 import pkg from '../package.json';
 import logo from './lib/logo';
 import checkVersion from './../check';
+import updateNotifier from 'update-notifier';
```
Comment (lukeed) [inline src/index.js]: Let's move this to line 2, so that it precedes all the local imports.

File: src/index.js [hunk 2]
```diff
@@ -31,3 +32,5 @@ yargs
 	.demandCommand()
 	.strict()
 	.argv;
+
+updateNotifier({pkg}).notify({ defer: true });
```
Comment (lukeed) [inline src/index.js]: This should be before all the `yargs(...)` stuff takes place -- aka, right after the `checkVersion()`.

Also, we don't need `defer:true` --- it's `true` by default.

File: src/index.js (hunk 1 above)
Comment (reznord) [reply src/index.js]: Done. Didn't think about this earlier.

File: src/index.js (hunk 2 above)
Comment (reznord) [reply src/index.js]: Done

File: src/index.js [hunk 3]
```diff
@@ -16,6 +17,8 @@ checkVersion();
 
 installHooks();
 
+updateNotifier({pkg}).notify({ defer: true });
```
Comment (lukeed) [inline src/index.js]: Can remove `defer:true` because [it is already the default](https://github.com/yeoman/update-notifier#defer). 👍

Comment (reznord) [reply src/index.js]: Ah! shoot

Comment (reznord) [reply src/index.js]: Done 🙏

**Pull request merged**